api/v1alpha1/ramenconfig_types.go

```diff
@@ -191,6 +191,10 @@ type RamenConfig struct {
 
 	// RamenOpsNamespace is the namespace where resources for unmanaged apps are created
 	RamenOpsNamespace string `json:"ramenOpsNamespace,omitempty"`
+
+	// StartDRPCReconciliationPaused is a flag to pause all the DRPC reconcile temporarily.
+	// This flag can be overridden by the annotation drplacementcontrol.ramendr.openshift.io/reconciliationUnpaused
+	StartDRPCReconciliationPaused bool `json:"startDRPCReconciliationPaused,omitempty"`
```

Comment:
The `Start...` here, is simply an indication that disabling is intended for new DRPCs only. But, we can drop it. I'll let Gowtham decides. Also, I like the word `Pause` instead of disable.

Reply:
I am not really certain we need this, to stop reconciliation just scale down the deployment to 0 replicas. That would scale down all controllers (i.e policy/cluster/etc.) but this seems excessive at present.

Once the annotation is removed for CG enabling, if a future use-case arises where a single controller from the set needs to be disabled, this maybe considered.